app/fastTrack/src/appRegistry/FastTrackBanner.js

```diff
@@ -1,20 +1,98 @@
-// @flow strict
+// @noflow
 
 import * as React from 'react';
-import { WithNativeNavigation } from '@kiwicom/mobile-shared';
-import { Translation } from '@kiwicom/mobile-localization';
+// eslint-disable-next-line
+import { View } from 'react-native';
+
+import {
+  SimpleCard,
+  Text,
+  CenteredView,
+  TextButton,
+  Modal,
+} from '@kiwicom/mobile-shared';
 
 // This will be prop types passed from native app
-type Props = {||};
+type Props = {|
+  +height: number,
+|};
+
+type State = {|
+  +isModalVisible: boolean,
+|};
+
+class FastTrackBanner extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      isModalVisible: false,
+    };
+  }
+
+  onOpenModal = () => {
+    this.setState({
+      isModalVisible: true,
+    });
+  };
+
+  onCloseModal = () => {
+    this.setState({
+      isModalVisible: false,
+    });
+  };
+
+  renderModal = () => (
+    <Modal
+      isVisible={this.state.isModalVisible}
+      onRequestClose={this.onCloseModal}
+      testId="fastTrackBanner"
+    >
+      <View>
+        <Text
+          style={{
+            marginBottom: 16,
+            textAlign: 'center',
+            fontWeight: 'bold',
+            color: '#006dc7',
+          }}
+        >
+          This is modal content.
+        </Text>
+        <TextButton
+          style={{ flexBasis: 100 }}
+          type="secondary"
+          onPress={this.onCloseModal}
+          title="Close modal"
+        />
+      </View>
+    </Modal>
+  );
 
-class FastTrackBanner extends React.Component<Props> {
   render() {
-    return <Translation passThrough="Put code here" />;
+    return (
+      <SimpleCard style={{ height: this.props.height }}>
+        <React.Fragment>
+          <Text
+            style={{
+              marginBottom: 16,
+              textAlign: 'center',
+              fontWeight: 'bold',
+              color: '#00a991',
+            }}
```

Comment:
I suggest you import `StyleSheet` from '@kiwicom/mobile-shared` and create styles, it is more performant than inline styles

Reply:
Also, you should `import { defaultTokens} from '@kiwicom/mobile-oribt` and use then for color and such